zulipterminal/model.py

```diff
@@ -423,7 +427,14 @@ def is_in_muted_topics(self, stream_id: int, topic: str) -> bool:
         """
         stream_name = self.stream_dict[stream_id]['name']
         topic_to_search = [stream_name, topic]  # type: List[str]
-        return topic_to_search in self.muted_topics
+        # The intent is to use the following data structure locally for lookups
+        # without having server version & feature level checks throughout the
+        # codebase.
+        muted_topics_lookup = [
+            [stream_name, topic]
+            for stream_name, topic, *_ in self.muted_topics
+        ]
+        return topic_to_search in muted_topics_lookup
```

Comment:
This approach does work, but it would be beneficial to have some defensive programming (assert on type of the incoming data), and I wonder if we can generate and use just one data-structure, like a dict?